src/pages/Home/Content/Content.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from "react";
+import { ContentStyle } from "./Content.style";
+import { FoodItem, Category } from "DummyData/DataTypes";
+import { mcSpicy } from "../../../DummyData/McSpicy";
+import { ovaltineMcFlurry } from "../../../DummyData/OvaltineMcFlurry";
+import { categories } from "../../../DummyData/Categories";
+import { SectionHeaderStyle } from "./SectionHeader.style";
+import ItemList from "./ItemList/ItemList";
+
+type Props = {};
+
+//Fetch ALlFoodItems from database
+const AllFoodItems: FoodItem[] = [
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  mcSpicy,
+  ovaltineMcFlurry,
+  ovaltineMcFlurry,
+  ovaltineMcFlurry,
+];
+
+const Content = React.forwardRef<HTMLDivElement, Props>(
+  ({}: Props, ref: React.ForwardedRef<HTMLDivElement>) => {
+    let toRenderSectionAndFoodItems: React.ReactNode[] = []; //This variable is returned.
+
+    categories.forEach((sectionHeader, indexOuter) => {
+      const foodItemToRender: { foodItem: FoodItem; index: number }[] = []; //foodItems for each section.
+
+      //check if foodItem category matches currentCategory, then add to foodItemToRender.
+      AllFoodItems.forEach((foodItem, indexInner) => {
+        foodItem.categories.forEach((foodItemCategory) => {
+          if (sectionHeader.name === foodItemCategory.name) {
+            foodItemToRender.push({
+              foodItem: foodItem,
+              index: indexOuter * 10 + indexInner,
```

Comment:
you don't need to create unique indexes like this. In the mapping function where you create the elements, just use the index. The key doesn't need to be unique throughout the app, it just needs to be unique in the mapping function that creates it. So if there are 3 categories, there can be 3 keys with index 0, as long as the index 0 keys are not created by the same map function

Reply:
fixed